common/api-review/storage.api.md

```diff
@@ -123,18 +168,17 @@ export class StorageService implements _FirebaseService {
     _delete(): Promise<void>;
     // @internal (undocumented)
     readonly _firebaseVersion?: string | undefined;
-    // (undocumented)
+    // @internal (undocumented)
     getAuthToken(): Promise<string | null>;
     // Warning: (ae-forgotten-export) The symbol "RequestInfo" needs to be exported by the entry point index.d.ts
     // Warning: (ae-forgotten-export) The symbol "Request" needs to be exported by the entry point index.d.ts
     //
     // @internal (undocumented)
     makeRequest<T>(requestInfo: RequestInfo_2<T>, authToken: string | null): Request_2<T>;

```

Comment:
s/makeRequest/_makeRequest and so on.

BTW, the "prune-dts" script enforces the underscore prefix convention and gets rid of the `@internal` tags.

Reply:
Added it as an extra step after api-extractor creates storage-public.d.ts, but there seems to be a little bug that cuts off the first comment (on CompleteFn). Can probably fix it later.